components/DesktopMenu/DesktopMenu.tsx

```diff
@@ -19,30 +24,34 @@ const DesktopMenu = () => {
 
   return (
     <div className="flex flex-row text-sm uppercase font-quicksand gap-x-12">
-      <div className="flex items-center pr-4">
-        <Link href="/status" target="_blank" rel="noreferrer">
-          <div className="font-bold cursor-pointer dark:text-white text-black">
-            Allowlist Status
-          </div>
-        </Link>
-      </div>
-      <div className="flex items-center font-bold font-quicksand">
-        <button
-          type="button"
-          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
-          onClick={() => onChangeThemeConfig("light")}
-        >
-          light
-        </button>
-        <ToggleButton onClick={onToggle} value={themeMode === "dark"} id="light_dark_switch" />
-        <button
-          type="button"
-          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
-          onClick={() => onChangeThemeConfig("dark")}
-        >
-          dark
-        </button>
-      </div>
+      {!isHidden && (
+        <div className="flex items-center pr-4">
+          <Link href="/status" target="_blank" rel="noreferrer">
+            <div className="font-bold cursor-pointer dark:text-white text-black">
+              Allowlist Status
+            </div>
+          </Link>
+        </div>
+      )}
+      {!isHidden && (
```

Comment:
Is there a reason this can't be moved to the above conditional block?

Reply:
will update soon.